### PR TITLE
hotfix/fix assay table rendering in ISA-Tab Viewer plugin

### PR DESCRIPTION
**Pull request by djcomlab**
Fixes multiple assay table rendering #270 in ISA-Tab Viewer visualization plugin.

Additionally cleans up the presentation a bit. Works very nicely in Galaxy scratchbook.

### CODE REVIEW

File: config/plugins/visualizations/isatab-viewer/static/js/ISATabViewer.js [hunk 1]
```diff
@@ -106,58 +104,52 @@ ISATabViewer.rendering = {
...
-            if (__ret != undefined) {
+        ISATabViewer.rawdatadict = rawdatadict;
+        let file_contents = ISATabViewer.rawdatadict[entrypoint];
+        let lines = file_contents.split(/\r\n|\r|\n/g);
+        let current_section = "";
+        let current_study;
+
+        for (let i = 0; i < lines.length; i++) {
+            let __ret = ISATabViewer.rendering.process_investigation_file_line(lines[i], current_study, current_section);
+            if (typeof __ret !== undefined) {
```
Comment (ilveroluca) [inline config/plugins/visualizations/isatab-viewer/static/js/ISATabViewer.js]: is it supposed to be `(typeof __ret !== 'undefined')`?

**Comment by pkrog** [comment]
Tested on minikube with MTBLS9. The two assays were visible.

**Comment by djcomlab** [comment]
@ilveroluca I think you mean inconsistent!

**Comment by ilveroluca** [comment]
Thanks for the correction :-)

**Comment by djcomlab** [comment]
I've made the correction, someone please review again...

**Pull request merged**